src/currency_protocol/currency_protocol_handler.inl

```diff
@@ -834,6 +837,9 @@ namespace currency
     LOG_PRINT_MAGENTA("TIME: network time difference is " << m_last_median2local_time_difference << " (max is " << TIME_SYNC_DELTA_TO_LOCAL_MAX_DIFFERENCE << ")", ((m_last_median2local_time_difference >= 3) ? LOG_LEVEL_2 : LOG_LEVEL_3));
     if (std::abs(m_last_median2local_time_difference) > TIME_SYNC_DELTA_TO_LOCAL_MAX_DIFFERENCE)
     {
+      // treat as error getting ntp time
+      if (m_disable_ntp)
+        return false;
```

Comment:
```suggestion
      {
        LOG_PRINT_RED("TIME: network time difference is " << m_last_median2local_time_difference << " (max is " << TIME_SYNC_DELTA_TO_LOCAL_MAX_DIFFERENCE << ") while NTP is disabled", LOG_LEVEL_0);
        return false;
      }
```

I'd like to make it a bit more verbose. Feel free to change/rephrase the log message.